protoc_plugin/lib/src/file_generator.dart

```diff
@@ -528,12 +526,24 @@ class FileGenerator extends ProtobufContainer {
 
   void writeBinaryDescriptor(IndentingWriter out, String identifierName,
       String name, GeneratedMessage descriptor) {
-    var descriptorText = base64Encode(descriptor.writeToBuffer());
+    var base64 = base64Encode(descriptor.writeToBuffer());
     out.println('/// Descriptor for `$name`. Decode as a '
         '`${descriptor.info_.qualifiedMessageName}`.');
+
+    var base64Lines = splitString(base64, 74).map((s) => "'$s'").join();
     out.println('final $_typedDataImportPrefix.Uint8List '
         '$identifierName = '
-        '$_convertImportPrefix.base64Decode(\'$descriptorText\');');
+        '$_convertImportPrefix.base64Decode($base64Lines);');
+  }
+
+  List<String> splitString(String str, int segmentLength) {
```

Comment:
```suggestion
  static List<String> _splitString(String str, int segmentLength) {
```

We could also move this to a utility library, along with the stuff in `string_escape.dart`. We may also have other fairly general utility functions elsewhere in the package.

Reply:
sgtm! (I may leave that as work for a future PR)

I in-lined your suggestion into my local edits